openwisp_radius/api/views.py

```diff
@@ -159,6 +160,23 @@ def get(self, request, *args, **kwargs):
 download_rad_batch_pdf = DownloadRadiusBatchPdfView.as_view()
 
 
+class UserLanguageMixin(object):
+    def save_language(self, user):
+        languages = self._parse_language_header(self.request)
```

Comment:
I tried to use https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.translation.get_language_from_request, but it always returns en-gb even if language headers passed is different.

Reply:
Instead of duplicating Django's internal code, which is never an acceptable solution, please use ipdb or any other debugging tool to find out what's going on when the django function is called: https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/utils/translation/trans_real.py#L515-L555

There must be an explanation for this.